### PR TITLE
Feature/mongoid 7 compatibility

### PR DESCRIPTION
**Pull request by jafuentest**
Added changes to readme and changelog on top of @Startouf changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## Requirements
 
-Mongoid 3, 4, 5 or 6.
+Mongoid 3, 4, 5, 6 or 6.
```
Comment (dblock) [inline README.md]: or 7

**Comment by dblock** [comment]
Looks like this repo isn't building in Travis either.

@sferik You're around? Lets move this into the mongoid org/make me admin/or enable CI?

File: README.md (hunk 1 above)
Comment (jafuentest) [reply README.md]: Ooops... Updated! xD

**Comment by dblock** [comment]
I've released 2.3.1 with Mongoid 7 support.

**Pull request merged**